lgc/CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/state)
     file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/state")
 endif()
 tablegen(LGC state/LgcDialect.cpp.inc -gen-dialect-defs --dialect lgc
+    -no-memory-effects # TODO: Remove this once all relevant LLVM branches have the new memory(...) attribute
```

Comment:
Please remove this change. It was added as a fix for the fact that some of the internal builds used a relatively old version of LLVM still. Actually, we should probably see about remove it in the internal version as well.

Reply:
Updated, thanks~